run_create_cloudwatch_dashboard.py

```diff
@@ -410,18 +426,19 @@ def run_create_cw_dashboard_alarm(name, settings):
         continue
 
     is_target_exists = True
-
-    if settings['TYPE'] == 'elasticbeanstalk':
-        run_create_cw_dashboard_elasticbeanstalk(settings['NAME'], settings)
-    elif settings['TYPE'] == 'rds/aurora':
-        run_create_cw_dashboard_rds_aurora(settings['NAME'], settings)
-    elif settings['TYPE'] == 'sqs,lambda,sms':
-        run_create_cw_dashboard_sqs_lambda_sms(settings['NAME'], settings)
-    elif settings['TYPE'] == 'alarm':
-        run_create_cw_dashboard_alarm(settings['NAME'], settings)
-    else:
-        print('"%s" is not supported' % settings['TYPE'])
-        raise Exception()
+if settings['TYPE'] == 'elasticbeanstalk':
+    run_create_cw_dashboard_elasticbeanstalk(settings['NAME'], settings)
+elif settings['TYPE'] == 'rds/aurora':
+    run_create_cw_dashboard_rds_aurora(settings['NAME'], settings)
+elif settings['TYPE'] == 'sqs,lambda,sms':
+    run_create_cw_dashboard_sqs_lambda_sms(settings['NAME'], settings)
+elif settings['TYPE'] == 'alarm':
+    run_create_cw_dashboard_alarm(settings['NAME'], settings)
+elif settings['TYPE'] == 'ramiel':
+    run_create_cw_dashboard_ramiel(settings['NAME'], settings)
+else:
+    print('"%s" is not supported' % settings['TYPE'])
+    raise Exception()
```

Comment:
@metocks // 1수준 들여쓰기 처리 부탁드립니다. 
```suggestion
    if settings['TYPE'] == 'elasticbeanstalk':
        run_create_cw_dashboard_elasticbeanstalk(settings['NAME'], settings)
    elif settings['TYPE'] == 'rds/aurora':
        run_create_cw_dashboard_rds_aurora(settings['NAME'], settings)
    elif settings['TYPE'] == 'sqs,lambda,sms':
        run_create_cw_dashboard_sqs_lambda_sms(settings['NAME'], settings)
    elif settings['TYPE'] == 'alarm':
        run_create_cw_dashboard_alarm(settings['NAME'], settings)
    elif settings['TYPE'] == 'ramiel':
        run_create_cw_dashboard_ramiel(settings['NAME'], settings)
    else:
        print('"%s" is not supported' % settings['TYPE'])
        raise Exception()
```